data/gui/widget/button_success.cfg

```diff
@@ -247,7 +247,7 @@
 					({GUI__BORDER_COLOR      ALPHA={ALPHA}})
 					({GUI__BORDER_COLOR_DARK ALPHA={ALPHA}}) ("21, 79, 109, 255") {IPF}}
 
-				{_GUI_BUTTON_TEXT (_"page^✔ Copied") ({FONT_SIZE}) () ("0, 255, 0, 255")}
+				{_GUI_BUTTON_TEXT (_"page^<span font_family='DejaVu Sans'>✔</span> Copied") ({FONT_SIZE}) () ("0, 255, 0, 255")}
```

Comment:
This needs a `#po:` explanation to the translations about why the font is switching.

Coding standards for `#po:` vs `# po:` although we don't have a set standard, 2/3 of the existing hints omit the space. When adding hints, I've worked on the basis of keeping it consistent within each campaign, for data/gui there aren't many hints at all, but omitting the space has a slight lead in data/gui, in addition to having the lead overall.

Reply:
Please if the hint message is adequate.

Reply:
Looks good